--- xalan-j_2_7_1/src/MANIFEST.MF.orig	2013-01-25 01:48:20.188703268 +0100
+++ xalan-j_2_7_1/src/MANIFEST.MF	2013-01-25 01:49:43.388695029 +0100
@@ -37,34 +37,3 @@ Implementation-Title: org.apache.xalan.x
 Implementation-Version: @impl.version@
 Implementation-Vendor: Apache Software Foundation
 Implementation-URL: http://xml.apache.org/xalan-j/
-
-Name: java_cup/runtime/ 
-Comment: Runtime component of JCup 
-Specification-Title: Runtime component of JCup
-Specification-Vendor: Princeton University 
-Specification-Version: 0.10k 
-Implementation-Title: runtime 
-Implementation-Version: @impl.version@
-Implementation-Vendor: Princeton University  
-Implementation-URL: http://www.cs.princeton.edu/~appel/modern/java/CUP/ 
-
-Name: org/apache/bcel/
-Comment: Byte Code Engineering Library 
-Specification-Title: Byte Code Engineering Library 
-Specification-Vendor: Markus Dahm 
-Specification-Version: 5.0 rc1 
-Implementation-Title: org.apache.bcel
-Implementation-Version: @impl.version@
-Implementation-Vendor: Apache Software Foundation
-Implementation-URL: http://jakarta.apache.org/bcel
-
-Name: org/apache/regexp/
-Comment: Java Regular Expression package 
-Specification-Title: Java Regular Expression package 
-Specification-Vendor: Apache Software Foundation 
-Specification-Version: 1.2
-Implementation-Title: org.apache.regexp
-Implementation-Version: @impl.version@
-Implementation-Vendor: Apache Software Foundation
-Implementation-URL: http://jakarta.apache.org/regexp
-
--- xalan-j_2_7_1/src/manifest.xsltc.orig	2013-01-25 01:48:20.189703256 +0100
+++ xalan-j_2_7_1/src/manifest.xsltc	2013-01-25 01:50:42.756976577 +0100
@@ -18,34 +18,3 @@ Implementation-Title: org.apache.xml
 Implementation-Version: @impl.version@
 Implementation-Vendor: Apache Software Foundation
 Implementation-URL: http://xml.apache.org/xalan-j/dist/
-
-Name: java_cup/runtime/
-Comment: Runtime component of JCup 
-Specification-Title: Runtime component of JCup
-Specification-Vendor: Princeton University 
-Specification-Version: 0.10k 
-Implementation-Title: runtime 
-Implementation-Version: @impl.version@
-Implementation-Vendor: Princeton University  
-Implementation-URL: http://www.cs.princeton.edu/~appel/modern/java/CUP/ 
-
-Name: org/apache/bcel/
-Comment: Byte Code Engineering Library 
-Specification-Title: Byte Code Engineering Library 
-Specification-Vendor: Markus Dahm 
-Specification-Version: 5.0 rc1 
-Implementation-Title: org.apache.bcel
-Implementation-Version: @impl.version@
-Implementation-Vendor: Apache Software Foundation
-Implementation-URL: http://jakarta.apache.org/bcel
-
-Name: org/apache/regexp/
-Comment: Java Regular Expression package 
-Specification-Title: Java Regular Expression package 
-Specification-Vendor: Apache Software Foundation 
-Specification-Version: 1.2
-Implementation-Title: org.apache.regexp
-Implementation-Version: @impl.version@
-Implementation-Vendor: Apache Software Foundation
-Implementation-URL: http://jakarta.apache.org/regexp
-
